tests/testlib/Makefile

```diff
@@ -21,7 +21,7 @@ all: libtests2n.so libtests2n.dylib
 include ../../s2n.mk
 
 CFLAGS += -I../../ -I../../api/ -I../../libcrypto-root/include/
-LDFLAGS += -L../../lib/ -lcrypto -lpthread -ls2n
+LDFLAGS += -L../../lib/ -L../../libcrypto-root/lib -lcrypto -lpthread -ls2n
```

Comment:
This shouldn't be necessary. We copy libcrypto.a into ../../lib/ and so should be linkable - and we do want to statically link against libcrypto. What error do you see without these options?